static/app/views/performance/browser/webVitals/pagePerformanceTable.tsx

```diff
@@ -214,7 +229,36 @@ export function PagePerformanceTable() {
   return (
     <span>
       <SearchBarContainer>
-        <SearchBar placeholder={t('Search for Pages')} onSearch={handleSearch} />
+        <StyledSearchBar
+          placeholder={t('Search for more Pages')}
+          onSearch={handleSearch}
+        />
+        <StyledPagination
+          pageLinks={pageLinks}
+          disabled={isProjectWebVitalsQueryLoading || isTransactionWebVitalsQueryLoading}
+          size="md"
+        />
+        {/* The Pagination component disappears if pageLinks is not defined,
+        which happens any time the table data is loading. So we render a
+        disabled button bar if pageLinks is not defined to minimize ui shifting */}
+        {!pageLinks && (
+          <Wrapper>
+            <ButtonBar merged>
+              <Button
+                icon={<IconChevron direction="left" size="sm" />}
+                size="md"
+                disabled
+                aria-label={t('Previous')}
+              />
+              <Button
+                icon={<IconChevron direction="right" size="sm" />}
+                size="md"
+                disabled
+                aria-label={t('Next')}
+              />
+            </ButtonBar>
+          </Wrapper>
```

Comment:
I tried the disabled prop and unfortunately it doesnt work because the `pagelinks` check sits above everything else in the pagination component. It's basically this code block that happens before everything lol:
```
  if (!pageLinks) {
    return null;
  }
```

Reply:
🤦🏻‍♂️ brutal. Could you reach out to FEs after GA to get this cleared up? It's _wild_ to have to fake this entire component just for a loading state